### DISCUSSION TITLE
Add nicknames from chat history to autocomplete

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5114] [Ideas]
I don't know if this is a bug or not, but I'm assuming it's not, since Twitch started showing chat history relatively recently.

In short, I suggest adding nicknames from chat history to autocomplete. Right now, only nicknames that have sent a message after the chat has loaded are autocomplete.

**Comment by pajlada** [comment]
Added by @Nerixyz in https://github.com/Chatterino/chatterino2/commit/075a7c5af035eac14be095b352c62acd4125f433
